lib/sqlalchemy/dialects/mysql/base.py

```diff
@@ -1766,7 +1768,21 @@ def visit_regexp_replace_op_binary(self, binary, operator, **kw):
 class MySQLDDLCompiler(compiler.DDLCompiler):
     def get_column_specification(self, column, **kw):
         """Builds column DDL."""
-
+        if (
+            self.dialect.is_mariadb is True
```

Comment:
**Federico Caselli** (CaselIT) wrote:

Done

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4869